src/main/java/com/zufar/onlinestore/payment/exception/PaymentIntentProcessingException.java

```diff
@@ -0,0 +1,14 @@
+package com.zufar.onlinestore.payment.exception;
+
+import lombok.Getter;
+
+@Getter
+public class PaymentIntentProcessingException extends RuntimeException{
+
+    private final String paymentMethodId;
+
+    public PaymentIntentProcessingException(String paymentMethodId) {
+        super(String.format("Payment intent with the paymentMethodID: %s cannot be processed.", paymentMethodId));
```

Comment:
`"Payment intent with the paymentMethodId = '%s' cannot be processed."`

Reply:
Done